### DISCUSSION TITLE
Custom icon in InfoLabel

### DISCUSSION THREAD

**Discussion by 15MariamS** [microsoft/fluentui#29790] [Q&A] [answered]
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

I'd like to add a custom icon instead of the `info` icon with the [InfoLabel component](https://react.fluentui.dev/?path=/docs/components-infolabel--default). Specifically, I want the icon to be a Sparkle, for AI-generated content.

### Have you discussed this feature with our team

Not yet

### Additional context

Here's what my own implementation looks like today:
<img width="270" alt="Screenshot 2023-11-07 at 2 08 15 PM" src="https://github.com/microsoft/fluentui/assets/81782111/58d96b98-de70-4fcc-8425-d4e30799f7a5">

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

Normal

**Comment by miroslavstastny** [comment, marked as answer]
You can override the default icon by passing it as a `children` to `infoButton` slot:
```jsx
import {
  Sparkle16Regular,
  Sparkle16Filled,
  bundleIcon
} from "@fluentui/react-icons";

//...

const Sparkle16 = bundleIcon(Sparkle16Filled, Sparkle16Regular);

//...

return (
  <InfoLabel
    info="This is example information for an InfoLabel."
    infoButton={{ children: <Sparkle16 /> }}
  >
    Example label
  </InfoLabel>
);
```

Keep in mind that if you override the icon, it will be your responsibility to use the correct size of the icon. `InfoLabel` will NOT scale the icon based on the `size` prop.
